### ISSUE TITLE
[Quest] Hell's Shells (ID: 25507) - Mount Hyjal

### ISSUE THREAD

**Issue by polymorphism98** [WoWManiaUK/Blackwing-Lair#4492]
**Links:** https://www.wowhead.com/quest=25507/hells-shells

**What is happening:**
1. Use the provided [item](https://www.wowhead.com/item=54744/heap-of-core-hound-innards) on the floor near Nemesis.
2. He comes and eats the item.
3. Right clicking (gear icon) on him, doesn't award the required [item ](https://www.wowhead.com/item=54745/nemesis-shell-fragment)to complete the quest.

**What should happen:**
1. Upon right clicking, the [item ](https://www.wowhead.com/item=54745/nemesis-shell-fragment)should be awarded.

**Is it crashing the server?:** No

**Other Information:** https://www.youtube.com/watch?v=oz1vmluVxT4

**Comment by Echinaceax** [comment]
Any reason why you closed this @polymorphism98 ?

**Comment by polymorphism98** [comment]
@Echinaceax - A while after reporting this issue, I found the required item in my bag, and was able to complete the quest. I am not able to recall if I got it completed by a GM or by some other means. But I do remember that the NPC is not lootable. If someone else can confirm the same, we can re-open this.

Is it possible for a tester to confirm? Let me know if its okay to reopen. Glad to do that.

Thanks.
